src/esp/gfx/GenericDrawable.cpp

```diff
@@ -113,6 +118,12 @@ void GenericDrawable::updateShaderLightingParameters(
 
 void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
                            Mn::SceneGraph::Camera3D& camera) {
+  if (!isRendererCreated) {
```

Comment:
Remove this if clause. Instead use CORRADE_ASSERT here to let it fail fast.

The if clause you put here just let the function return silently, without any further info given back to the developer.

Developer may just get a black screen and spend even more time to debug.